mongodb-datastore/deploy/mongodb-datastore.yaml

```diff
@@ -35,11 +35,11 @@ spec:
         - containerPort: 8080
         resources:
           requests:
-            memory: "128Mi"
+            memory: "256Mi"
             cpu: "100m"
           limits:
-            memory: "256Mi"
-            cpu: "1000m"
+            memory: "512Mi"
```

Comment:
does the mongodb datastore really require more resources or is this a leftover from debugging?

Reply:
the in-cluster debugging was not working properly in some cases - increasing the memory limit helped in that case